docs/Documentation/tutorial.md

```diff
@@ -1,23 +1,25 @@
+The **VIC-RBM** model is a coupled model that links three sub-models together: the **VIC** model; the routing model; and the **RBM** model. The **VIC** model is a hydrologic model that uses meteorological forcing data as input and simulates hydrologic variables such as runoff, evapotranspiration, and soil moisture, at each grid cell. The routing model takes the output from the **VIC** model along with flow network information as input, and output streamflow at specified locations along stream network. The **RBM** model takes both the streamflow results and the meteorological data as input and calculate stream temperature along stream network. Thus, the **VIC-RBM** model as a whole is able to simulate both streamflow and stream temperature at spatial scales determined by the basic **VIC** gridded network configuration. **VIC** gridded networks have been developed for the Continental United States (CONUS) at 1/16, 1/8, 1/4, 1/2 and 1 degree of latitude and longitude. As presently configured, the **VIC-RBM** model system simulates daily-averaged or subdaily stream temperatures in accordance with the timestep of input.
 ## 2. Model input
 ### 1)Input for the VIC hydrologic model
 *  Meteorological forcing data at each grid cell
     *  Minimum requirement:
         - daily precipitation
         - daily maximum and minimum temperature
         - wind speed
-    *  VIC implements algorithms which are able to generate a full set of meteorological variables (see model output section for a list of meteorological variables) at daily or sub- daily time steps. These variables can also be directly given as input.
+    *  VIC includes a meteorological data disaggregator that calculates subdaily meteorological variables required by VIC from the minimum forcing variables. Details can be seen in the [VIC website](http://vic.readthedocs.io/en/develop/Documentation/Drivers/Classic/ForcingData/).
 * Soil properties at each grid cell
 * Vegetation information at each grid cell
 ### 2)Input for the routing model
 *  Flow direction file – describes the topology of the river basin network.
-*  Unit hydrograph file - contains the grid cell impulse response function.
+*  Unit hydrograph file - contains the grid cell impulse response function. <br />
```

Comment:
Thanks for pointing this out. I add a sentence  in the `Input for the routing model` section:
`flow velocity, diffusivity and flow distance values - we set them to spatially constant values so don't have separate input files for these)`